js/base-ui.js

```diff
@@ -134,35 +134,53 @@ function baseUi () {
 	 *
 	 * @param dataArray options to choose from
 	 * @param dataTitle title for the window
-	 * @param messageCountIncomplete message when user does not choose correct number of choices
 	 * @param displayFormatter function to format dataArray for display
-	 * @param count exact number of  options the user must choose
+	 * @param count exact number of  options the user must, mutually exclusive with countMin and countMax
+	 * @param countMin lowest number of options the user must choose, requires countMax, mutually exclusive with count
+	 * @param countMax highest number of options the user must choose, requires countMax, mutually exclusive with count
+	 * @param additionalHTML additional html code, such as a button
 	 * @param note add a note at the bottom of the window
+	 * @param messageCountIncomplete message when user does not choose correct number of choices
 	 * @return {Promise}
 	 */
-	d20plus.ui.chooseCheckboxList = async function (dataArray, dataTitle, {displayFormatter = null, count = null, note = null, messageCountIncomplete = null} = {}) {
+	d20plus.ui.chooseCheckboxList = async function (dataArray, dataTitle, {displayFormatter = null, count = null, countMin = null, countMax = null, additionalHTML = null, note = null, messageCountIncomplete = null} = {}) {
 		return new Promise((resolve, reject) => {
+			// Ensure count, countMin, and countMax don't mess up
+			// Note if(var) is false if the number is 0. countMin is the only count allowed to be 0
+			isInt = Number.isInteger;
```

Comment:
please... don't randomly alias things for no reason. Long lines are happy lines.
also, you've made this a global variable, so we have now polluted global scope with a `window.isInt` for the rest of the script's execution